docs/whats-new/zowe-v3-migration.md

```diff
@@ -0,0 +1,121 @@
+# Zowe V3 Migration Guide
+
+This guide outlines the steps and changes required to migrate from Zowe v2 to Zowe v3. While the migration process is similar to a minor release upgrade, there are several new and updated configuration parameters to consider. Follow the steps below to ensure a smooth migration.
+
+
+## Prerequisites
+
+Before starting the migration, ensure the following system requirements are met:
+
+- **z/OSMF**: Version V2R5 with APAR PH12143 is required.
+- **Java**: Ensure java.home is set to **Java 17**.
+- **Node.js**: Ensure node.home is set to **Node.js 18+**.
+- **Keyrings**: If you are using keyrings, verify that you use safkeyring:// instead of safkeyring:////.
+
+
+
+## Configuration changes
+
+### New Configuration
+
+**components.zaas**:  
+  Previously part of the components.gateway component (used as the auth service), zaas has now been moved to a separate component.  
+  If you do not explicitly configure this section, zaas will still be enabled by default, using **port 7558**.
+
+### Updated Configuration Parameters
+
+#### Gateway z/OSMF Service ID
+
+The service ID for gateway zosmf has changed to **ibmzosmf**. If you are using zosmf as your auth service, this needs to be updated.
+
+```yaml
+components:
+  gateway:
+    apiml:
+      security:
+        auth:
+          zosmf:
+             jwtAutoconfiguration: jwt
```

Comment:
also, "auto" has been removed.
a user must either choose "jwt" (default) or "ltpa"
both rocket and ibm are having trouble using jwt, and are using "ltpa".